src/WebAPI/SignalR/RulesHub.cs

```diff
@@ -66,6 +67,11 @@ public async Task BroadcastMessage(string user, string message)
         CancellationToken cancellationToken
     )
     {
+        if (Context.User.IsAuthenticated())
+        {
+            //TODO: Track user stats
```

Comment:
Same here - a link to the PBI shows developers that see this code that there is a task allocated for it and that someone is working on it.

In practical terms here that is probably not necessary as you're immediately picking this up and working on it. Definitely worth doing for longer term technical debt though.

Reply:
Noted, thanks for the tip @matt-goldman 🙂